posts/2021/07/numpy-accessibility-guidelines.md

```diff
@@ -0,0 +1,180 @@
+accessible technology. In fact, they feared that they didn’t know enough. Yet
+they strongly believed that accessible technology is a right every person should
+access and that NumPy could be accessible too.
+
+This small team started with Melissa Weber Mendonça and me, Mars Lee. Our
+previous skills and projects varied from working on documentation, front-end
+code, grant-writing, community work, software maintenance, and consulting work.
+
+We were very inspired by JupyterLab's big strides forward in accessibility this
+year. In the open-source ecosystem, accessibility is often considered
+low-priority, scary or someone else’s job. If JupyterLab could do it, so could
+NumPy. We decided to take this on.
+
+Making NumPy accessible is something that should have started a long time ago.
+However it  never quite started, mostly due to a lack of knowledge. Not everyone
+knows what accessible technology is, who it benefits, what guidelines to use or
+how to implement them. With all these gaps in knowledge, it can be intimidating
+to even start this conversation.
+
+This blog post is the first in a series of three, as accessibility is a big
+topic. Additionally, a note on terminology- the terms ‘users with a disability’
+and ‘disabled user’ are used interchangeably, and some people may prefer one
+term over the other.
+
+**Who needs accessibility? **
+
+![A comic about how everyone benefits from accessibility. The example is about
+having a situational disability and being provided accommodations to overcome
+it.](/images/2021/07/numpy-accessibility/situation_disability.png) [_External
+link to the above illustration's full
+transcription_](https://gist.githubusercontent.com/MarsBarLee/4f5f30ede8631dec8fd9fcd0526aa065/raw/6ab7e61cec2dc173a84b04cec246fde6b2841d81/situation_disability.png)
+
+First, who needs accessibility? Your first answer might be ‘for people with
+disabilities’ but ultimately everyone benefits from having accessible
+technology.
+
+This is because accessibility is about making technology accessible to the full
+range of human experience. Disabilities can affect sight, hearing, cognition and
+physical motion. 
+
+Through this lens, we’ve all been at least situationally disabled. We’ve all
+been to websites with small, hard to read text. Maybe you’ve had a broken mouse
+and could only navigate through the keyboard. Or needed to watch a video but had
+no working speakers.
+
+We have felt the frustration of the world not working the way you need it to. We
+have felt relief in finding commonly built-in solutions, such as increasing our
+browsers' text size, turning on captions or learning to switch tabs with
+keyboard shortcuts.
+
+
+![A comic about how different people respond to the same barrier. The example
+here is about graphs with colors that are inaccessible for people with
+deuteranopia.](/images/2021/07/numpy-accessibility/disability_not_recognized.png)
```

Comment:
It's thoughtful that you are being specific and referring to what kind of color blindness, but I'm not sure everyone knows what it is. Maybe it makes sense to say something like`for people with deuteranopia, red-green color blindness`?